exec.go

```diff
@@ -11,7 +11,7 @@ import (
 	"golang.org/x/net/context"
 )
 
-func runCmd(ctx context.Context, cancel context.CancelFunc, cmd *exec.Cmd) {
+func runCmd(ctx context.Context, cancel context.CancelFunc, cmd *exec.Cmd) int {
```

Comment:
went with another solution -- returning from the runCmd doesn't work for `go runCmd(...`